File: westcoast/src/main/java/frc/commands/targeting/FindTarget.java
package frc.commands.targeting;

import edu.wpi.first.wpilibj2.command.CommandBase;
import frc.subsystems.DriveTrainSubsystem;
import org.jetbrains.annotations.NotNull;


public final class FindTarget extends CommandBase {
    @NotNull
    private final DriveTrainSubsystem drive;

    public FindTarget(@NotNull DriveTrainSubsystem drive) {
        this.drive = drive;
    }

    @Override
    public void execute() {
        drive.tank(-0.35, 0.35, false);
    }

    @Override
    public boolean isFinished() {
        return drive.vision.hasTarget();
    }
}

File: westcoast/src/main/kotlin/frc/commands/targeting/FindTarget.kt
package frc.commands.targeting

import edu.wpi.first.wpilibj2.command.CommandBase
import frc.subsystems.DriveTrainSubsystem

class FindTarget(private val drive: DriveTrainSubsystem) : CommandBase() {
    override fun execute() {
        drive.tank(-0.35, 0.35, false)
    }

    override fun isFinished(): Boolean {
        return drive.vision.hasTarget
    }
}